pages/docs/manual/latest/pipe-last.mdx

```diff
@@ -0,0 +1,51 @@
+# Pipe Last
+
+<Warn>
+We discourage the usage of the `|>` / pipe-last operator in BuckleScript codebases. Using it may sometimes have a runtime cost due to currying.
```

Comment:
i forgot to put a newline between the `<Warn>` and the text in my original comment... this prevents MDX from rendering the content as markdown...

so it would be required to do:

~~~
<Warn>

We discourage the usage [...]

</Warn>
~~~

Sorry 😅

Reply:
@ryyppy 👍 No worries, fixing as suggested. I actually never worked on MDX files before. I learn new things every day which is good.